cnd/src/btsieve/bitcoin/bitcoind_connector.rs

```diff
@@ -10,6 +10,7 @@ use serde::Deserialize;
 #[derive(Copy, Clone, Debug, Deserialize)]
 pub struct ChainInfo {
     bestblockhash: BlockHash,
+    #[serde(with = "crate::btsieve::bitcoin::serde_bitcoind_network")]
```

Comment:
Given that we don't need this functionality anywhere else, can we just have a local function instead of whole new module?

We only need `deserialize` so you can use `deserialize_with` here: https://serde.rs/field-attrs.html#deserialize_with

Reply:
Can be addressed in a follow-up if you don't want to do it now although I would appreciate if we don't have ugly identifiers like this :)

Reply:
It's not that I don't want to but rather that I didn't know how to do this :D

Reply:
I think I got it, rebase + push in a minute.

Reply:
Good on you, you learned something about serde :)